source/base/TimeSpan.ooc

```diff
@@ -111,6 +109,7 @@ TimeSpan: cover {
 	hours: static func (count: Double) -> This { This new(DateTime ticksPerHour * count) }
 	days: static func (count: Double) -> This { This new(DateTime ticksPerDay * count) }
 	weeks: static func (count: Double) -> This { This new(DateTime ticksPerWeek * count) }
+	fromNanoseconds: static func (nanoseconds: Long) -> This { This new(nanoseconds / DateTime nanosecondsPerTick) }
```

Comment:
or perhaps I should just rename the nanoseconds one, since it would be a breaking change... *sigh*